client/db_test.go

```diff
@@ -245,3 +245,29 @@ func ExampleDB_Insecure() {
 	// Output:
 	// aa=1
 }
+
+func ExampleDB_Open() {
```

Comment:
@bdarnell, the problem is I didn't know what to test for. Normally, I'd write a `Test*` and simply check if it panics, but @tamird adviced to put it in `db_test.go` with other examples. I've purposedly named it `ExampleDB_Open` (not `Example_Open`) so it won't show up in docs just like `ExampleDB_Insecure` above.